cmd/entrypoint/runner.go

```diff
@@ -31,7 +31,10 @@ import (
 	"syscall"
 
 	"github.com/tektoncd/pipeline/pkg/entrypoint"
-	"github.com/tektoncd/pipeline/pkg/pod"
+)
+
+const (
+	TektonHermeticEnvVar = "TEKTON_HERMETIC"
```

Comment:
This one is duplicated between here and `pkg/pod` or did we move it from `pkg/pod` to here ? 
Ideally, if it's duplicated, we would need to move it to a `constant` package that we can import without having other packages imported.

Reply:
Yeah it is duplicated
When you said we need to move it to a `constant` package you mean under `pkd/pod/constants` ?